spec/spec_helper.rb

```diff
@@ -14,3 +20,24 @@
     c.syntax = :expect
   end
 end
+
+class TestServiceProvider
+  include ::ActiveModel::Model
+  include ::ActiveModel::Validations
+  include ::ActiveRecord::Validations
+  include IdentityValidations::ServiceProviderValidation
+
+  attr_accessor :friendly_name, :issuer, :ial, :redirect_uris, :failure_to_proof_url, :push_notification_url, :saml_client_cert
+
+  def initialize(**args)
+    super
+  end
+
+  def new_record?
+    true
+  end
+
+  def self._reflect_on_association(foo)
```

Comment:
I would call the argument `_` if it's unused

Reply:
good call